Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java

```diff
@@ -78,34 +80,58 @@ public String getContent() {
     try {
       content = (String) this.jsonObject.get("content");
       Document jsoupDoc = Jsoup.parse(content);
-      content = jsoupDoc.text();
+      content = jsoupDoc.toString();
     } catch (Exception e) {
       Log.e("getContent_Error:", e.getMessage(), e);
       content = null;
     }
     return content;
   }
 
-  public Integer getWordCount() {
-    Integer wordCount = 0;
+  public int getWordCount() {
+    int wordCount;
     try {
-      wordCount = (Integer) this.jsonObject.get("word_count");
+      long longWordCount = (long) this.jsonObject.get("word_count");
+      wordCount = (int) longWordCount;
     } catch (Exception e) {
       Log.e("getWordCount_Error:", e.getMessage(), e);
-      wordCount = null;
+      wordCount = 0;
     }
     return wordCount;
   }
 
   /**
+   * Bring date of contents
+   *
+   * <p> Need to be fixed. (return null)
+   *
+   * <p> In my estimation, it might be API prob or dateformat prob
+   *
+   * @return string of published date with 'yyyy-MM-dd' format
+   */
+  public String getDate() {
+    String date;
+    try {
+      SimpleDateFormat parserSDF = new SimpleDateFormat("yyyy-MM-dd'T'KK:mm:ss"); //2016-09-30T07:00:12.000Z yyyy-MM-dd'T'KK:mm:ss
+      Date datePublished = parserSDF.parse((String) this.jsonObject.get("date_published"));
+      SimpleDateFormat newDate = new SimpleDateFormat("yyyy-MM-dd");
+      date = newDate.format(datePublished);
+    } catch (Exception e) {
+      Log.e("getData_Error:", e.getMessage(), e);
+      date = null;
+    }
+    return date;
+  }
+
+  /**
    * Makes possible to use {@link #getJsonResponse(String pageUrl)}
    *
    * <p> Sending request and receiving response via API is impossible in main Thread.
    *
    * <p> AsyncTask runs in the background of main Thread. AsyncTask = Thread + Handler
    *
    * @returns json format response from Mercury-API (detailed response is described in {@link
-   * #getJsonResponse(String strUrl)} document.
+   *     #getJsonResponse(String strUrl)} document.
```

Comment:
https://google.github.io/styleguide/javaguide.html#s7.1.3-javadoc-at-clauses

I just followed the javadoc.